src/main/java/com/joyent/manta/cosbench/MantaStorage.java

```diff
@@ -192,11 +223,8 @@ public void deleteContainer(final String container, final Config config) {
     }
 
     @Override
-    public void createObject(final String container,
-                             final String object,
-                             final InputStream data,
-                             final long length,
-                             final Config config) {
+    public void createObject(final String container, final String object, final InputStream data, final long length,
```

Comment:
Long argument lists should be broken out on multiple lines.

Reply:
Okay, changed it in my IDE so it will format long args to multiple lines.